packages/database/test/info.test.ts

```diff
@@ -132,6 +132,8 @@ describe('.info Tests', function() {
   });
 
   it.skip('database.goOffline() / database.goOnline() connection management', function() {
+    // NOTE: getFreshRepo() no longer takes a hostname, so this test needs to be reworked.
+    // Need to figure out how to re-enable this test. Also see comment #27-#32.
```

Comment:
What does `comment #27-#32` refer to?

Reply:
https://github.com/firebase/firebase-js-sdk/blob/d93e15307e400443c23467c6646e5cd104380834/packages/database/test/info.test.ts#L27-L36

Done. I guess there's not much value to put such reference in comments.